odfuzz/entities.py

```diff
@@ -109,6 +109,8 @@ def _append_corresponding_queryable(self, query_group):
 class DirectBuilder:
     """A class for building and initializing all queryable entities with metadata passed in constructor."""
     def __init__(self, metadata, restrictions,method):
+        if method not in ["GET","DELETE","PUT","POST"]:
+            raise ValueError("The http method value \'{}\' is invalid\nUse either GET, DELETE, PUT or POST".format(method))
```

Comment:
```suggestion
            raise ValueError("The HTTP method \'{}\' is invalid\nUse either GET, DELETE, PUT or POST".format(method))
```

Reply:
Enum class would handle that "other input" by default. But since this is the outer-facing API, if it more practical to have String (e.g. this code is for example called in Java-Python stack), then leave it as string with this check.